### ISSUE TITLE
A simple example of library, please

### ISSUE THREAD

**Issue by amahlaka** [max90727/RA8835_Arduino-Uno#2]
Hi! i am having little problem understanding how to use this library, i would love a short example with maybe a basic hello world script
thanks

**Comment by max90727** [comment]
Hi: 
It was a bit long time and I am not working on that any more. I will try to find out the example. Thanks

**Comment by amahlaka** [comment]
here is my device: http://www.datasheetlib.com/datasheet/1140063/wg320240c_winstar-display.html
and here is my current code: 
`
 #include <RA8835_lib.h>

SED1335 lcd = SED1335();
void setup()
{
  lcd.GLCD_InitializePorts();
 lcd.GLCD_Initialize();

}

void loop()
{

  lcd.GLCD_createCircle(4 ,1 ,3 ,2);
lcd.GLCD_createCircle(5 ,3 ,6 ,7);
lcd.GLCD_TextGoTo(2,2);
lcd.GLCD_WriteText("a");

}
 `
now just wondering the wiring, how can Read and wirte be in analog inputs?

**Comment by max90727** [comment]
Hi:
I found my old project. Here is the code I did, I don't have a lcd by my side, so I cannot test the code.
// Do not remove the include below
#include "Arduino_SED1335.h"
RA8835 lcd;
void setup() {
// Add your initialization code here
    //Initializing LCD
    lcd.GLCD_Initialize();
    //lcd.setInverse(true);
    lcd.GLCD_ClearGraphic();
    lcd.GLCD_ClearText();
    //Initializing SD
    Serial.begin(9600);
    while (!Serial);
    pinMode(10, OUTPUT);
    if (!SD.begin()) {
        lcd.GLCD_TextGoTo(0, 0);
        lcd.GLCD_WriteText("initialization failed!");
        return;
    }
    lcd.GLCD_TextGoTo(0, 0);
    lcd.GLCD_WriteText("initialization done.");
    lcd.GLCD_Bitmap("240320/1.bmp", 0, 0, 240, 320);

}

// The loop function is called in an endless loop
void loop() {

//  delay(1000);
//  lcd.GLCD_Bitmap("240320/2.bmp", 0, 0, 240, 320);
//  delay(1000);
//  lcd.GLCD_Bitmap("240320/3.bmp", 0, 0, 240, 320);
//  delay(1000);
//  lcd.GLCD_Bitmap("240320/4.bmp", 0, 0, 240, 320);
//  delay(1000);
//  lcd.GLCD_Bitmap("240320/5.bmp", 0, 0, 240, 320);
//  delay(1000);
}

**Comment by max90727** [comment]
I don't think I used the circle thing, maybe not work.

**Comment by amahlaka** [comment]
What is the pin 10 used for?

la 6. hel 2016 10.21 Jianyuan Ma notifications@github.com kirjoitti:

**Comment by max90727** [comment]
I am not sure, could be the back light.

**Comment by amahlaka** [comment]
Do you have the bitmaps as well?

**Comment by max90727** [comment]
Yes. They were read from the SD card. I think that pin 10 is the pin for the SD break board.

**Comment by amahlaka** [comment]
can you send the bitmaps?

**Comment by max90727** [comment]
I am sorry, I did this for a company two year ago, those bitmaps are theirs and I don't have them.

**Comment by amahlaka** [comment]
ok, i noticed that when i wire everythying to arduino, the voltage regulator starts to get really hot, do i need a external PS for display,, it also has own input for logic and lcd power

**Comment by max90727** [comment]
No. The back light is only 4~5v, <1A. Seems like your LCD doesn't have that input. Usually they are pin 19 and 20 for the LCD.
